third_party/move/move-model/src/model.rs

```diff
@@ -3256,6 +3321,12 @@ pub struct FunctionData {
     pub(crate) transitive_closure_of_called_funs: RefCell<Option<BTreeSet<QualifiedId<FunId>>>>,
 }
 
+impl FunctionData {
```

Comment:
Why do we need this to be public on FunctionData? Typically this is accessed via FunctionEnv (where I think it already exists)

Reply:
Assuming you're talking about `FunctionData::is_inline()`, there was a moment where I was iterating through `FunctionData` and reconstructing the `FunctionEnv` just to query `is_inline` seemed silly.  Now I don't seem to need it anymore.

So, we don't need it.  Removed.